HeliPort/Credits.rtf

```diff
@@ -2,13 +2,13 @@
 \cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fswiss\fcharset0 Helvetica-Bold;\f2\fnil\fcharset0 HelveticaNeue;
 }
 {\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red0\green0\blue0;\red0\green0\blue233;
-}
+\red0\green0\blue0;}
 {\*\expandedcolortbl;;\cssrgb\c0\c0\c0\cname textColor;\csgray\c0\c0;\cssrgb\c0\c0\c93333;
-}
+\cssrgb\c0\c0\c0\cname textColor;}
```

Comment:
`textColor` is already defined, so you can safely revert the changes from L5-12.